src/contributors/04-api.md

```diff
@@ -44,9 +44,15 @@ https://lemmy.ml/api/{version}/comment/like
 
 ### RSS/Atom feeds
 
-- All - `/feeds/all.xml?sort=Hot`
-- Community - `/feeds/c/community-name.xml?sort=Hot`
-- User - `/feeds/u/user-name.xml?sort=Hot`
+| Type            | url                                    |
+| --------------- | -------------------------------------- |
+| All             | `/feeds/all.xml?sort=Hot`              |
+| Local           | `/feeds/local.xml?sort=Hot`            |
+| Community       | `/feeds/c/community-name.xml?sort=Hot` |
+| User            | `/feeds/u/user-name.xml?sort=Hot`      |
+| Your front page | `/feeds/front.xml/{jwt_token}`         |
+| Your inbox      | `/feeds/inbox.xml/{jwt_token}`         |
+| Your modlog     | `/feeds/modlog.xml/{jwt_token}`        |
```

Comment:
Might be worth linking the available sort types, and how to obtain jwt token. Or at least mention that fully working feed links can be obtained via web ui.

Reply:
> Or at least mention that fully working feed links can be obtained via web ui.

I'll do this.